frontend/src/components/Landing.vue

```diff
@@ -31,7 +32,7 @@ import TreeLogs from '@/assets/images/tree-logs.jpg';
                     class="landing-button"
                     outlined
                     disabled
-                    label="Login with BCeID"
+                    :label="`Login with ${IdpProvider.BCEIDBUSINESS}`"
```

Comment:
Replaced the hard-coded text to the enums, so it can be easily changed in the future.